.gitignore

```diff
@@ -19,3 +19,4 @@ node_modules/
 .settings
 
 *.tgz
+/Dockerfile
```

Comment:
What's this about?

Reply:
oops, that was me building a local image for testing pre-release builds of the SDK.